esp/esp/themes/theme_data/bigpicture/templates/contact.html

```diff
@@ -0,0 +1,61 @@
+{% extends "main.html" %}
+{# TODO(benkraft): Many of these changes should go in the non-theme version #}
```

Comment:
Really it should just get merged into the main `contact.html`, I just need to check it doesn't break any other themes.  I'll just do that now as a separate PR.